transients_nearby_galaxies/transients_nearby_galaxies.ipynb

```diff
@@ -0,0 +1,1182 @@
+{
```

Comment:
<p>Could we add some of the transients to this plot? It would be cool to see a visual representation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I decided to make this an exercise, because it would involve a few steps that I think would otherwise break the flow of the notebook.</p>

Reply:
<p>Good point. We should stay focused on the purpose of the tutorial.</p><p> </p>